libs/community/langchain_community/document_loaders/pebblo.py

```diff
@@ -406,6 +408,10 @@ def _get_app_details(self) -> App:
             runtime=runtime,
             framework=framework,
             plugin_version=PLUGIN_VERSION,
+            client_version=Framework(
+                name="langchain_community_version",
+                version=langchain_community.__version__,
+            ),
```

Comment:
Is it necessary to send both `langchain` version in the `framework` field and `langchain_community` version in `client_version`? Do we need both?

Reply:
Currently, it is being used in pebblo report and DAXA UI, to show the version, but eventually it should update with langchain_community_version.
Let @srics confirm whether we should update/remove this

Reply:
https://github.com/dristysrivastava/langchain/blob/6c43012f509705015066cf26e52656106e41343a/libs/community/langchain_community/document_loaders/pebblo.py#L409